workshop/Lab5/README.md

```diff
@@ -0,0 +1,239 @@
+# Lab 5 Deploy an OSB service and attach it to a deployment
+
+Learn to provision a service instance, create a binding for your service instance,
+and use that binding to connect your deployment to that service.
+
+# Before we begin
+
+If you skipped Lab0 or otherwise didn't create a Persistant Volume, we need one for Minibroker
```

Comment:
If we haven't needed this until now, why do it so early on?

Reply:
link to lab0

Reply:
there's a link to lab 0 like literally 1 line from where you linked.

Reply:
Let's just assume they did the previous labs and remove this section.  Supporting the idea of people jumping into the middle of all of this labs will be a pain to support.